File: tests/CoreTests/AnonymousObjects/ClrAnonymousObjects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using SharpKit.JavaScript;

namespace CoreTests.AnonymousObjects
{
    [JsType(JsMode.Prototype, NativeJsons = false)]
    public class NonNativeJsons
    {
        public void F()
        {
            var xx = new object();
            var x = new { a = "b" };
            var a = x.a;
        }
    }
}

File: tests/CoreTests/AnonymousObjects/ClrAnonymousObjects.js
;
if (typeof(CoreTests) == "undefined")
    var CoreTests = {};
if (typeof(CoreTests.AnonymousObjects) == "undefined")
    CoreTests.AnonymousObjects = {};
CoreTests.AnonymousObjects.NonNativeJsons = function (){
};
CoreTests.AnonymousObjects.NonNativeJsons.prototype.F = function (){
    var xx = new System.Object.ctor();
    var x = $CreateAnonymousObject({
        a: "b"
    });
    var a = x.get_a();
};

